railties/test/generators/plugin_generator_test.rb

```diff
@@ -106,11 +106,28 @@ def test_generating_test_files_in_full_mode_without_unit_test_files
     end
   end
 
-  def test_generating_adds_dummy_app_in_full_mode_without_sprockets
-    run_generator [destination_root, "-S", "--full"]
+  def test_generator_if_skip_sprockets_is_given
+    run_generator [destination_root, "--skip-sprockets"]
```

Comment:
No, this test hasn't had any relations to `--full` option, so this is extra here.